Git-commit: b49eab40eb02773b8422ef870cab2642a305a4ab
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 13 Jan 2021 21:27:56 +0000
Subject: [PATCH] SQUASH: Revert "overlays: Fix dtc warnings in i2c-gpio"
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

This reverts commit 1c15edc0dca002c8536e9f1f5e1ec43017815018.

This revert and its neighbour are opposites. When squashing, delete
the original commits as well.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts b/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
index 2323a61edf07..63231b5d7c0c 100644
--- a/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
@@ -11,9 +11,6 @@ fragment@0 {
 		target-path = "/";
 
 		__overlay__ {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
 			i2c_gpio: i2c@0 {
 				reg = <0xffffffff>;
 				compatible = "i2c-gpio";
-- 
2.31.1

